src/main/java/com/google/sps/servlets/MealServlet.java

```diff
@@ -85,13 +123,36 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         return;
     }
 
+    private Entity createMealEntity(Meal meal) {
+        Entity mealEntity = new Entity("Meal");
+        mealEntity.setProperty("id", meal.getId());
+        mealEntity.setProperty("title", meal.getTitle());
+        mealEntity.setProperty("description", meal.getDescription());
+        mealEntity.setProperty("ingredients", meal.getIngredients());
+        mealEntity.setProperty("type", meal.getType());
+ 
+        return mealEntity;
+    }
+
+    private void getMealList(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        Query query = new Query("Meal").addSort("id", SortDirection.DESCENDING);
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        PreparedQuery results = datastore.prepare(query);
+
+        List<Meal> meals = getDataFromDatastore(results);
+        Gson gson = new Gson();
+        response.setContentType("application/json;");
+        response.getWriter().println(gson.toJson(meals));
+        return;
+    }
+
     private void searchMeal(HttpServletRequest request, HttpServletResponse response) throws IOException {
         // Fuction returns hardcoded result
         // TODO(sandatsian): implement search algorithm here for MVP
-        String gson = new Gson().toJson(dishes);
+        /*String gson = new Gson().toJson(dishes);
```

Comment:
Probably you do not need to comment anything, just delete - you always can find it in previous commits.